packages/buckram/assets/styles/components/specials/_contributors.scss

```diff
@@ -18,64 +18,65 @@
     text-align: $contributors-text-align;
   }
 
-  .contributor__type {
-    margin-top: $contributors-margin-top;
-    margin-bottom: $contributors-margin-bottom;
-  }
-
-  // TODO: replace clearfix hack with better CSS Grid solution when Prince supports grid layout
-  .contributor__name__and__links:after {
-    content: "";
-    display: table;
-    clear: both;
-  }
+  .contributor {
+    &__type {
+      margin-top: $contributors-margin-top;
+      margin-bottom: $contributors-margin-bottom;
+    }
+    // TODO: replace clearfix hack with better CSS Grid solution when Prince supports grid layout
+    &__name__and__links:after {
+      content: "";
+      display: table;
+      clear: both;
+    }
 
-  .contributor__profile__picture {
-    float: $contributor-profile-picture-float;
-    margin-top: if-map-get($contributor-profile-picture-margin-top, $type);
-    margin-right: $contributor-profile-picture-margin-right;
-    margin-bottom: $contributor-profile-picture-margin-bottom;
-    margin-left: $contributor-profile-picture-margin-left;
-    width: 33%;
-    max-width: 400px;
-    border-radius: $contributor-profile-picture-border-radius;
-  }
+    &__profile__picture {
+      float: $contributor-profile-picture-float;
+      margin-top: if-map-get($contributor-profile-picture-margin-top, $type);
+      margin-right: $contributor-profile-picture-margin-right;
+      margin-bottom: $contributor-profile-picture-margin-bottom;
+      margin-left: $contributor-profile-picture-margin-left;
+      width: 33%;
+      max-width: 400px;
+      border-radius: $contributor-profile-picture-border-radius;
+    }
 
-  .contributor__name {
-    font-family: $h1-font-family;
-    font-size: $h3-font-size;
-    text-indent: $contributors-text-indent;
-  }
+    &__name {
+      font-family: $h1-font-family;
+      font-size: $h3-font-size;
+      text-indent: $contributors-text-indent;
+    }
 
-  .contributor__institution {
-    font-family: $h1-font-family;
-    text-transform: $contributor-institution-text-transform;
-    text-indent: $contributors-text-indent;
-    margin: 0;
-  }
+    &__institution {
+      font-family: $h1-font-family;
+      text-transform: $contributor-institution-text-transform;
+      text-indent: $contributors-text-indent;
+      margin: 0;
+    }
 
-  .contributor__website {
-    text-indent: $contributors-text-indent;
-  }
+    &__website {
+      text-indent: $contributors-text-indent;
+    }
 
-  .contributor__links {
-    margin-top: if-map-get($contributor-links-margin-top, $type);
-    display: inline-block;
+    &__links {
+      margin-top: if-map-get($contributor-links-margin-top, $type);
+      display: inline-block;
 
-    svg {
-      width: 45px;
-      height: 45px;
-    }
-    .contributor__link {
-      &__icon {
-        text-decoration: none;
+      svg {
+        width: 45px;
+        height: 45px;
+      }
+      .contributor__link {
```

Comment:
should this become `&__link` for consistency's sake @arzola ?

Reply:
In that case we probably would need to rename the class of those <a> to `contributor__links__link`

Reply:
The thing is we are targeting a child there not the same class

Reply:
I sent a little change what do you think?